content/en/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -201,7 +201,7 @@ For example: `partition in (customerA, customerB),environment!=qa`.
 
 ### LIST and WATCH filtering
 
-LIST and WATCH operations may specify label selectors to filter the sets of objects
+LIST and {{< glossary_tooltip text="WATCH" term_id="watch" >}} operations may specify label selectors to filter the sets of objects
```

Comment:
```suggestion
For **list** and **watch** operations, you can specify label selectors to filter the sets of objects
```

Reply:
ok . I'll remove the glossary tooltip from here as well.